infra/src/main/java/com/redhat/service/smartevents/infra/metrics/MetricsService.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.service.smartevents.infra.metrics;
+
+import java.util.Set;
+
+public interface MetricsService<T> {
```

Comment:
I made the `MetricsService` in both `manager` and `operator` implement a common interface.

Previously we had two implementations that had different APIs and usage.